pkgs/games/legendary-gl/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, python38Packages
+}:
+
+with python38Packages;
+
```

Comment:
Remove this
```suggestion
```

Reply:
I added this because the app requires Py >= 3.8. What's a better way?

Reply:
Got it, using `python38Packages.callPackage` in all-packages :)